Formula/rio.rb

```diff
@@ -0,0 +1,24 @@
+class Rio < Formula
+  desc "Kubernetes based MicroPaaS"
+  homepage "https://rio.io"
+  url "https://github.com/rancher/rio.git",
+    :using    => :git,
+    :tag      => "v0.1.1-rc3",
```

Comment:
We cannot ship release candidates. Software must be stable.

In addition, after making changes please squash and for push with the commit message reading:

```
rio <version> (new formula)
```

Reply:
Changed to `0.1.0`